pkg/policy/mapstate.go

```diff
@@ -1162,8 +1167,10 @@ func (ms *mapState) denyPreferredInsertWithChanges(newKey Key, newEntry MapState
 	// Sanity check on the newKey
 	if newKey.TrafficDirection >= trafficdirection.Invalid.Uint8() {
 		stacktrace := hclog.Stacktrace()
```

Comment:
nit: Don't really need this var. Might as well the make the call in the log line directly.